packages/extension/test/unit/extension_spec.js

```diff
@@ -1,136 +0,0 @@
-require('../spec_helper')
-
-let { exec } = require('child_process')
-let fs = require('fs-extra')
-const eol = require('eol')
-const path = require('path')
-const Promise = require('bluebird')
-const extension = require('../../index')
-
-const cwd = process.cwd()
-
-fs = Promise.promisifyAll(fs)
-exec = Promise.promisify(exec)
-
-describe('Extension', () => {
-  context('.getCookieUrl', () => {
-    it('returns cookie url', () => {
-      expect(extension.getCookieUrl({
-        name: 'foo',
-        value: 'bar',
-        path: '/foo/bar',
-        domain: 'www.google.com',
-        secure: true,
-      })).to.eq('https://www.google.com/foo/bar')
-    })
-  })
```

Comment:
It doesn't look like there's a test for this anymore, unless I missed it.

Reply:
I'm not really sure of a great place to put it or if it makes sense to port over, but the code is getting tested alongside the automation client in an integration fashion significantly. 

A lot of the cookie class methods/utilities aren't unit tested but integration tested, and I'm hoping to test them further once we refactor the logic out of `@packages/server` since the methods are used circularly.